### PR TITLE
Es/implement locators

### PR DESCRIPTION
**Pull request by relaxing-observer**

### CODE REVIEW

File: tests/pages/main_page_locators.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+class LogInPopupLocators:
+    CONFIRMATION_LOG_IN_BUTTON = (by.xpath("//form[@data-ref='login_modal']//button[@type='submit']"))
+    EMAIL_FORM = (by.xpath("//input[@placeholder='email@email.com']"))
+    PASSWORD_FORM = (by.xpath("//input[@placeholder='Password']"))
+
+
+class MainPageLocators:
+    ACCEPT_COOKIES_BUTTON = (by.xpath("//div[@id='cookie-popup-with-overlay']//button"
+                                      "[@class='cookie-popup-with-overlay__button']")
+                             )
```
Comment (TA-user) [inline tests/pages/main_page_locators.py]: consider using triple quotes if it is needed

File: tests/pages/main_page_locators.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    INPUT_DESTINATION_FORM = (by.xpath("//input[@id='input-button__destination']"))
+    LAST_ITEM_DROPDOWN_AIRPORT = (by.xpath("(//span[@data-ref='airport-item__name'])[last()]"))
+    RETURN_FORM = (by.xpath("//fsw-input-button[@uniqueid='dates-to']"))
+    SEARCH_FLIGHT_BUTTON = (by.xpath("//button[@data-ref='flight-search-widget__cta']"))
+
+
+class SearchHotelsTabLocators:
+    CHECK_IN_FORM = (by.xpath("//hp-input-button[@uniqueid='check-in']"))
+    CHECK_OUT_FORM = (by.xpath("//hp-input-button[@uniqueid='check-out']"))
+    FIRST_ITEM_HOTEL_DROPDOWN = (by.xpath("//icon[@iconid='glyphs/destination-pin']"))
```
Comment (TA-user) [inline tests/pages/main_page_locators.py]: consider using css if it is possible

**Review by TA-user — CHANGES_REQUESTED** [review]

File: tests/pages/main_page_locators.py (hunk 1 above)
Comment (relaxing-observer) [reply tests/pages/main_page_locators.py]: Fixed

File: tests/pages/main_page_locators.py (hunk 2 above)
Comment (relaxing-observer) [reply tests/pages/main_page_locators.py]: It was decided by team to use a mixed locators strategy. We decided to use locators from previous project (with bdd approach). It is possible to use css, but to reduce the time I have used locators on xpath. Should I change all old locators?

**Review by kit-91-daniil — APPROVED** [review]

**Review by yefim-mironyuk — APPROVED** [review]

**Review by TA-user — APPROVED** [review]

**Pull request merged**